generate_job_parameters_file_PHiLiP.sh

```diff
@@ -1,10 +1,11 @@
 # This file quickly generates the job paramters file
 
 filename="my_job_parameters_file.sh"
-time="2:00:00" # wall time
+walltime="2:00:00" # wall time
 job_name="run_parameters_file" # slurm job name
 nodes=1 # number of nodes
-ntasks_per_node=40 # number of processors per node; refer to https://docs.computecanada.ca/wiki/Advanced_MPI_scheduling
+ntasks_per_node=16 # number of processors per node; refer to https://docs.computecanada.ca/wiki/Advanced_MPI_scheduling
+memory_per_node="63G" # requested memory per node; NOTE: Must correspond to ntasks_per_node; refer to https://docs.computecanada.ca/wiki/Advanced_MPI_scheduling
```

Comment:
Maybe rephrase comment to `NOTE: Must correspond to n_tasks_per_node * memory_per_core` to reduce confusion for new users

Reply:
good call, its been fixed